### PR TITLE
85 add discord instantiation

### LINKED ISSUE #85: Add Discord instantiation
**Short description:**
Send Discord getters in function of the userID and fix some routes.

**Describe the solution you'd like**
A function that take an userID as a parameter and send back all the guilds where the user is Administator.
Add JWT protection to Discord getters routes.
Add Discord getters routes in the DB.

### PR DESCRIPTION
**Pull request by tbellicha**
# Purpose

The purpose of this pull request is to list all available guilds where the user is Administrator.

Fixes #85 

## Updated features
* JWT protection to Discord getters
* Add `guilds` scope to Discord

## Fixed feature
* Fix the getAvailableGuilds which now returns all guilds where the user is Administrator using his Discord Token

### CODE REVIEW

**Review by LucasTesnier — DISMISSED** [review]

**Review by TomDUVAL-MAHE — DISMISSED** [review]
NTR

File: server/server_app.js [hunk 1]
```diff
@@ -665,14 +666,48 @@ app.post(
 /**
  * @brief List all available Guilds where the bot is.
  */
-app.get(
+app.post(
```
Comment (ZQUEMA) [inline server/server_app.js]: If you get data, the method to use should be GET.

File: server/server_app.js [hunk 2]
```diff
@@ -665,14 +666,48 @@ app.post(
...
+      console.log(err)
+      return res.status(400).send('An error occured.')
+    }
+  }
+)
+
+/**
+ * @brief List available performers, such as bot/user.
+ */
+app.post(
```
Comment (ZQUEMA) [inline server/server_app.js]: If you get data, the method to use should be GET.

**Review by ZQUEMA — CHANGES_REQUESTED** [review]
Change the methods used to get data

**Review by ZQUEMA — DISMISSED** [review]

**Review by ZQUEMA — APPROVED** [review]

**Review by LucasTesnier — APPROVED** [review]

**Pull request merged**